src/Utilities/SystemsHelpers.js

```diff
@@ -9,8 +9,9 @@ import { packageSystemsColumns } from '../SmartComponents/Systems/SystemsListAss
 import { defaultCompoundSortValues } from './constants';
 import { patchSetDetailColumns } from '../SmartComponents/PatchSetDetail/PatchSetDetailAssets';
 import { InsightsLink } from '@redhat-cloud-services/frontend-components/InsightsLink';
+import isEmpty from 'lodash/isEmpty';
 
-export const buildFilterConfig = (search, filter, apply, osFilterConfig) => ({
+export const buildFilterConfig = (search, filter, apply, osFilterConfig, groupsFilterConfig, featureFlag) => ({
```

Comment:
Neatpick: it is better to name the feature flag explicitly as it belongs to the group feature only. For example, **isGroupsEnabled**